packages/cashier/src/components/crypto-fiat-converter/crypto-fiat-converter.tsx

```diff
@@ -171,21 +164,6 @@ const CryptoFiatConverter = observer(
                                 classNameHint='crypto-fiat-converter__hint'
                                 data-testid='dt_converter_to_amount_input'
                             />
-                            {is_timer_visible && (
-                                <Timer
-                                    onComplete={() => {
-                                        const from_rate = getRate(from_currency || '');
-                                        const to_rate = getRate(to_currency || '');
-                                        const converted_amount = (Number(converter_from_amount) * to_rate) / from_rate;
-                                        onChangeConverterFromAmount(
-                                            { target: { value: converter_from_amount } },
-                                            from_currency,
-                                            to_currency,
-                                            converted_amount
-                                        );
-                                    }}
-                                />
-                            )}
```

Comment:
since we are subscribing to exchange_rates endpoint , we no longer need the timer to request fr the latest update from server

Reply:
yes @george-usynin-deriv :
https://deriv-group.slack.com/archives/CEP7GPTP0/p1697015066756969?thread_ts=1696216625.554429&cid=CEP7GPTP0

https://deriv-group.slack.com/archives/C023SRL88VB/p1699331239840629